app/recurring_job.go

```diff
@@ -38,6 +38,7 @@ const (
 	HTTPClientTimout = 1 * time.Minute
 
 	SnapshotPurgeStatusInterval = 5 * time.Second
+	SnapshotPurgeStatusTimeout  = 1 * time.Hour
```

Comment:
Is 1 hour enough for a large volume purge? I am not sure what the best value is here...

Reply:
Since we're unsure of the appropriate timeout value, I've set it to 24 hours with a comment.